clash-lib/src/Clash/Netlist/Types.hs

```diff
@@ -280,6 +280,7 @@ data Declaration
   | InstDecl
       EntityOrComponent                  -- FIELD Whether it's an entity or a component
       (Maybe Comment)                    -- FIELD Comment to add to the generated code
+      (Maybe [Attr'])                    -- FIELD Attributes to add to the generated code
```

Comment:
Maybe this should be `[Attr']`? Seems like `Just []` ~ `Nothing`.

Reply:
Good point. See the latest commit.